scripts/config-split/templates/config_split.config_split.env.yml.twig

```diff
@@ -0,0 +1,14 @@
+uuid: {{ uuid }}
```

Comment:
I worry about what will happen if/when config split updates their configuration schema, and we also need to update this template to match.

I'm not sure of a better alternative though.